queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
+      where: >-
+        (kind == 'path' and zoom < 15) or
+        (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 16
+      properties: [name, ref, network]
+      where: >-
+        (kind == 'path' and zoom < 17) or
+        (kind in ['minor_road', 'rail'] and zoom < 15) or
+        (kind == 'major_road' and zoom < 12) or
+        (kind == 'highway' and zoom < 7)
```

Comment:
I think we've already crossed the Rubicon for having a fully general tile set - dropping information in return for fewer / more compact features seems desirable given that perhaps 99% of the requests that we'll serve might come from our app or someone's app displaying something closely related to tangram-skin-and-bones.

_However_, I think there's a need for a way to do more customized tilesets, if only so that we can generate version N and N+1 simultaneously and go back and forth between them at will. Infrastructure that means that small changes are small and we can do multiple deploys per week, rather than having a 3-day turn-around time for a prod deploy. Not wanting to sound like a broken record, but I think the foundation of that is NEAT tiles.